doc/Type/X/Cannot/Empty.pod6

```diff
@@ -0,0 +1,26 @@
+=begin pod :kind("Type") :subkind("class") :category("exception")
+
+=TITLE class X::Cannot::Empty
+
+=SUBTITLE Error due to inappropriate usage of an empty collection
+
+    class X::Cannot::Empty is Exception { }
+
+Error thrown (or wrapped in a C<Failure>) when inappropriately using an empty
+collection.
```

Comment:
Can you add an example or two please?

Reply:
Good point. I'll add an example where a routine fails with an `X::Cannot::Empty` if the collection is empty.